Resources/views/CRUD/delete.html.twig

```diff
@@ -15,6 +15,12 @@ file that was distributed with this source code.
     {% include 'SonataAdminBundle:CRUD:action_buttons.html.twig' %}
 {%- endblock -%}
 
+{% block action_menu %}
+    {% if action is defined %}
+        {{ knp_menu_render(admin.sidemenu(action), {'currentClass' : 'active', 'template': sonata_admin.adminPool.getTemplate('action_menu_template')}, 'twig') }}
+    {% endif %}
+{% endblock %}
```

Comment:
1. Should be linebroken
2. There is a lot of repetition… why not define this block in a dedicated template, and [use](http://twig.sensiolabs.org/doc/tags/use.html) it ?